README.md

```
@@ -169,6 +172,39 @@ media HTTP server configuration:
   `true` will make the server enforce its cipher list ordering for TLS version
   up to 1.2, defaults to `false`
 
+## Ironic Networking
```

markdownlint-cli2 (0.22.0)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)